### PR TITLE
Update to jobs beta

### PR DESCRIPTION
**Pull request by jdmcd**

### CODE REVIEW

**Comment by vkill** [comment]
Hi @mcdappdev 

Please set eventLoopGroup threads to 2 in [JobsRedisDriverTests.swift](https://github.com/vapor/jobs-redis-driver/blob/master/Tests/JobsRedisDriverTests/JobsRedisDriverTests.swift#L145) , then you can see precondition failed at [EventLoop.preconditionInEventLoop](https://github.com/apple/swift-nio/blob/2.9.0/Sources/NIO/EventLoop.swift#L582) .

**Comment by jdmcd** [comment]
@vkill Good catch, that is weird. @tanner0101 Any ideas here?

File: Sources/JobsRedisDriver/JobsRedisDriver.swift [hunk 1]
```diff
@@ -4,38 +4,40 @@ import NIO
...
+    
+    /// The logger
+    let logger: Logger
     
     /// Creates a new `RedisJobs` instance
     ///
     /// - Parameters:
     ///   - client: The `RedisClient` to use
-    public init(client: RedisClient) {
+    public init(client: RedisClient, eventLoopGroup: EventLoopGroup) {
```
Comment (vkill) [inline Sources/JobsRedisDriver/JobsRedisDriver.swift:22]: @mcdappdev  We should ensure worker's EventLoop equal to redis connection's EventLoop.

Maybe below changes are better.

Change `init(client: RedisClient, eventLoopGroup: EventLoopGroup)` to `init(client: ConnectionPool<RedisConnectionSource>)`
Change `func get(key: String, eventLoop: JobsEventLoopPreference)` to `func get(key: String, eventLoop: EventLoop)`
And change `self.client.rpoplpush` to `self.client.withConnection(eventLoop: .delegate(on: eventLoop)) { redis in redis.rpoplpush }`

Comment (jdmcd) [reply Sources/JobsRedisDriver/JobsRedisDriver.swift:22]: That would involve changing the Jobs protocol which is solid currently, I think there needs to be another solution. @tanner0101 Thoughts?

**Comment by tanner0101** [comment]
Merging this into master to work on it further, thanks!

**Comment by jdmcd** [comment]
Awesome, thanks!

**Pull request merged**